app/main.py

```diff
@@ -1,25 +1,65 @@
-from abc import ABC
+class IntegerRange:
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(
+            self, owner: "SlideLimitationValidator",
+            name: str) -> None:
```

Comment:
If there are no arguments on the same line as the opening parenthesis, the closing parenthesis should not have any arguments on the same line as well. Apply this change to all related places